src/app/screens/Onboard/ConnectRaspiBlitz/index.tsx

```diff
@@ -16,10 +16,17 @@ export default function ConnectRaspiBlitz() {
   const [loading, setLoading] = useState(false);
 
   function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
-    setFormData({
-      ...formData,
-      [event.target.name]: `https://${event.target.value.trim()}:8080`,
-    });
+    if (event.target.name === "url") {
```

Comment:
I'd say whenever you have a function that consists of one big`if` that's an indication that the function should be split up. (generally it's good to avoid `if` as much as possible).
This makes the function smaller, doing less and thus also easier to test. 

so in this case write a `handleUrl` or something that only handles the URL and use a different function for the macaroon. 
This makes it also less likely to silently break if somebody potentially changes the input name (from url to something eles). - your code would fail silently and it would not be noticed.

Reply:
Sounds good, I thought about making 2 functions for this but wasn't sure if that was the right thing to do. Will update now!